recipes/selectfasta/meta.yaml

```diff
@@ -0,0 +1,26 @@
+{% set name = "selectfasta" %}
+{% set version = "2.0" %}
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  git_url: https://github.com/andvides/selectFasta.git
```

Comment:
Better create a released version of the software and reference the tarball here.